optaplanner-docs/src/main/docbook/en-US/Chapter-Benchmarking_and_tweaking/Chapter-Benchmarking_and_tweaking.xml

```diff
@@ -974,6 +974,49 @@
       </section>
     </section>
 
+    <section xml:id="statisticalBenchmarking">
+      <title>Statistical Benchmarking</title>
+
+      <para>If you want to minimize the influence of your environment on the benchmark results, you
+        can configure the number of times each single benchmark run is repeated. The results of
+        those runs will get statistically aggregated and are even visible in the report
+        individually.</para>
+
+      <figure>
+        <title>Sub Single Benchmark Summary Statistic</title>
+
+        <mediaobject>
+          <imageobject>
+            <imagedata fileref="images/Chapter-Benchmarking_and_tweaking/subSingleBenchmarkSummaryStatistic.png"/>
+          </imageobject>
+        </mediaobject>
+      </figure>
+
+      <para>To configure this in your benchmarks, add a <literal>&lt;subSingleCount&gt;</literal>
+        element to a <literal>&lt;solverBenchmark&gt;</literal> element:</para>
+
+      <programlisting language="xml">&lt;plannerBenchmark&gt;
+  ...
+  &lt;solverBenchmark&gt;
+    &lt;name&gt;...&lt;/name&gt;
+    &lt;solver&gt;
+      ...
+    &lt;/solver&gt;
+    &lt;subSingleCount&gt;10&lt;subSingleCount&gt;
+  &lt;/solverBenchmark&gt;
+  ...
+&lt;/plannerBenchmark&gt;</programlisting>
+
+      <para>You can also configure <literal>subSingleCount</literal> in the
+        <link linkend="inheritedSolverBenchmark"><literal>&lt;inheritedSolverBenchmark&gt;</literal></link> element
+        and possibly overwrite it in the individual <literal>&lt;solverBenchmark&gt;</literal> elements.</para>
+
+      <note>
+        <para><literal>subSingleCount</literal> is by default set to 1. Any positive integer is a
+          valid value.</para>
+      </note>
```

Comment:
Might be noting the obvious? We probably already have too many note and warning elements in the docs already. Maybe just add "(defaults to 1)" somewhere in the paragraphs above - and I might not even mention it's a positive integer (as the example code makes that obvious - is a count ever negative? - if a user is distracted enough to put a negative number there, he'll get a clear error message anyway).

PS: It's a strictly positive integer, not a positive integer because 0 isn't allowed :)